resources/external_control.urscript

```diff
@@ -95,14 +110,73 @@ thread speedThread():
   stopj(5.0)
 end
 
+thread jointTrajectoryThread():
+  textmsg("Executing trajectory")
+  textmsg(trajectory_points_left)
+  while trajectory_points_left > 0:
+    textmsg("Waiting to read point")
+    enter_critical
+    #reading trajectory point + blend radius + type of point (cartesian/joint based)
+    raw_point = socket_read_binary_integer(TRAJECTORY_DATA_DIMENSION+1+1, "trajectory_socket")
+    trajectory_points_left = trajectory_points_left - 1
+    exit_critical
+    if raw_point[0] > 0:
+      q = [ raw_point[1] / MULT_jointstate, raw_point[2] / MULT_jointstate, raw_point[3] / MULT_jointstate, raw_point[4] / MULT_jointstate, raw_point[5] / MULT_jointstate, raw_point[6] / MULT_jointstate]
+      tmptime = raw_point[7] / MULT_time
+      blend_radius = raw_point[8] / MULT_time
+
+      textmsg("Executing Point")
+      textmsg(trajectory_points_left)
+      textmsg(q)
+      textmsg(tmptime)
+      textmsg(blend_radius)
+      textmsg(raw_point[9])
```

Comment:
Should we leave all of these in? In general might be worth thinking about reducing text output, but should not be critical.

Reply:
good point, I'll clean this up a bit.

Reply:
Removed in 7d8236d